src/loader/endpoints/GoogleDocsLib.py

```diff
@@ -0,0 +1,46 @@
+import pickle
```

Comment:
this should be in a utils file.

Let's save the `/endpoints/*` to actual endpoints.